src/main/resources/static/js/common.js

```diff
@@ -262,17 +262,18 @@ function unlockSubscriptionButtons() {
 }
 
 function cropUsername(username) {
-    var usernameMaxLength = 15;
+    var numberOfDotsAfterUsernameOnTooLongName = 3;
+    var ellipses = "...";
 
-    if (username.length <= usernameMaxLength) {
+    if (username.length <= getUsernameMaxDisplayLength()) {
         return username;
     }
 
-    var breakUsernameAtIndex = usernameMaxLength - 3;
-    return username.substr(0, breakUsernameAtIndex) + "...";
+    var breakUsernameAtIndex = getUsernameMaxDisplayLength() - numberOfDotsAfterUsernameOnTooLongName;
```

Comment:
Maybe it could just use ellipses.length? :D

Reply:
Done.